README.md

```diff
@@ -134,6 +134,18 @@ jobs:
     location: 'east us'
 ```
 
+# Local Development and Testing
+
+If you wish to develop and test changes against a local fork or development repo, you can do so by including the `node_modules` in tagged release branch. Note that the `aci-deploy` repository does not include these modules in the master branch, so you cannot point your action to `aci-deploy/master` to pick up recent commits. 

```

Comment:
I think these are generic and we can also include in Azure/Actions guidelines if it is not present

Reply:
@kanika1894 you can refer to this link here directly for [developer-guildelines](https://github.com/Azure/actions/blob/main/docs/developer-guildelines.md).

Reply:
@BALAGA-GAYATRI as discussed, are the guidelines updated to focus on the node_modules part?